src/utils.ts

```diff
@@ -245,3 +252,81 @@ export function validateCoverageMetrics(
 
     return failures;
 }
+
+function getFormattedMetricData(coverageData: StatsFieldDataType) {
+    const { skipped, covered, pct } = coverageData;
+    return {
+        missed: { ...DEFAULT_METRIC_VALUE.missed, value: skipped },
+        covered: { ...DEFAULT_METRIC_VALUE.covered, value: covered },
+        percentage: { ...DEFAULT_METRIC_VALUE.percentage, value: pct },
+    };
+}
+
+function getAggregatedMetrics(metricsList: any[]) {
+    const aggData: any = {
+        lines: { total: 0, covered: 0, skipped: 0 },
+        branches: { total: 0, covered: 0, skipped: 0 },
+        functions: { total: 0, covered: 0, skipped: 0 },
+    };
+
+    metricsList.forEach((metric: any) => {
+        const { lines, functions, branches } = metric;
+
+        aggData.lines.total += lines.total;
+        aggData.lines.covered += lines.covered;
+        aggData.lines.skipped += lines.skipped;
+
+        aggData.functions.total += functions.total;
+        aggData.functions.covered += functions.covered;
+        aggData.functions.skipped += functions.skipped;
+
+        aggData.branches.total += branches.total;
+        aggData.branches.covered += branches.covered;
+        aggData.branches.skipped += branches.skipped;
+    });
+
+    for (const key: string in aggData) {
+        const { total, covered } = aggData[key];
+
+        let pct: number = (covered / total) * 100;
+        if (!Number.isInteger(pct)) {
+            pct = parseFloat(pct.toFixed(2));
+        }
+        aggData[key].pct = pct;
+    }
+
+    return aggData;
+}
+
+export function exposeCoverageMetrics(coverageMetrics: Record<string, any>[], projectRoot: any): void {
+    const moduleData: any[] = [];
```

Comment:
Can we type this since we know what the type each entry in `moduleData` looks like?